database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -24,7 +24,7 @@
 /**
  * This class implements an array-like collection on top of a Firebase location.
  */
-class FirebaseArray implements ChildEventListener {
+public class FirebaseArray implements ChildEventListener {
```

Comment:
I reverted to making it no modifier. If this is the case, I think all of `FirebaseArray`'s methods should also be no modifier since `FirebaseArray` will never be used outside of the package.